searchcore/src/vespa/searchcore/proton/documentmetastore/lid_gid_key_comparator.cpp

```diff
@@ -7,15 +7,15 @@ namespace proton::documentmetastore {
 LidGidKeyComparator::LidGidKeyComparator(const document::GlobalId &gid,
                                          const MetaDataStore &metaDataStore)
     : _gid(gid),
-      _metaDataStore(metaDataStore),
+      _metaDataView(&metaDataStore.acquire_elem_ref(0)),
```

Comment:
Consider adding a function on MetaDataStore that returns a view (e.g. make_read_view returning vespalib::ConstArrayRef < RawDocumentMetaData >) to make the usage clearer.

Reply:
Done.